src/app/lib/synchronization.coffee

```diff
@@ -33,28 +33,30 @@ module.exports = class Synchronization
             log.info 'start synchronization'
             @currentSynchro = true
 
-            @checkFilter (err) =>
+            #@checkFilter (err) =>
+            #    return @_finishSync err, callback if err
+
+            @checkFirstReplication (err) =>
                 return @_finishSync err, callback if err
 
-                @checkFirstReplication (err) =>
+                @syncCozyToAndroid live: false, (err) =>
```

Comment:
Does this method be specific to android? Else, it may be better to rename the method to something like `syncCozyToMobile`